Extension/src/LanguageServer/copilotCompletionContextProvider.ts

```diff
@@ -123,8 +129,19 @@ export class CopilotCompletionContextProvider implements ContextResolver<CodeSni
             const client = clients.getClientFor(docUri);
             if (!client) { throw WellKnownErrors.clientNotFound(); }
             const getCompletionContextStartTime = performance.now();
+
+            // Start collection of project traits concurrently with the completion context computation.
+            const telemetryProperties: Record<string, string> = {};
+            const telemetryMetrics: Record<string, number> = {};
+            const projectContextPromise = (async (): Promise<ProjectContext | undefined> => {
+                const getProjectContextStartTime = performance.now();
+                const projectContext = await getProjectContext(docUri, { flags: {} }, telemetryProperties, telemetryMetrics);

```

Comment:
I think it's fine to pass a hardcoded `flags: {}` here, but just wanted to double check that 1) that will pass only the language standard trait and 2) we don't expect to need to experiment with any other configuration for traits before we ship a new pre-release version. I believe both of those conditions are true.

Reply:
yes to both questions. In particular, the flags are not present anymore since the original API being used is now gone.